src/EnergyPlus/HVACManager.cc

```diff
@@ -515,6 +518,8 @@ namespace HVACManager {
 
 			ManageEMS( emsCallFromEndSystemTimestepBeforeHVACReporting, anyEMSRan ); // EMS calling point
 
+			CheckMassBalances();
```

Comment:
It shouldn't add any significant time to the simulation, and we should always be checking for mass conservation problems. There bay be a better time to check these. Ideas?